docs/src/repl_scripts.md

```diff
@@ -1,3 +1,33 @@
+# REPL script (debugging workflow example)
+
+```{julia}
+julia> ]
+pkg> activate .buildkite
+```
+
+```julia
+julia> using Revise # See also: Infiltrator.jl and Main.@infiltrate
```

Comment:
I don't think Revise is a dependency of ClimaAtmos - how could this work as a test?

Reply:
Switching to use `Literate.jl` should allow lines with # to be treated as comments and run all other lines as standard julia code, so we can still exercise the `AtmosConfig, AtmosSimulation, atmos_solve!(simulation)` and `step!(simulation.integrator)` methods - Revise is still an external reference as a doc example.